app/consumers/job_creator_consumer.rb

```diff
@@ -1,12 +1,15 @@
 # frozen_string_literal: true
 
-require 'lib/notifications'
-require 'lib/dispatcher'
+require 'notifications'
+require 'dispatcher'
 
 class JobCreatorConsumer < Racecar::Consumer
   subscribes_to Notifications::INCOMING_TOPIC
 
   def process(kafka_message)
+    message_value = kafka_message.value
+    Rails.logger.debug("Received message: #{message_value}")
+
     message = Message.from_json(kafka_message.value)
```

Comment:
Why not reuse the previously extracted `message_value` here?

Reply:
That was the intention. Forgot about it, sorry.